src/main/java/org/mitre/synthea/export/TextExporter.java

```diff
@@ -263,18 +518,42 @@ private static void encounter(List<String> textRecord, Encounter encounter) {
    * @param condition
    *          The condition to add to the export
    */
-  private static void condition(List<String> textRecord, Entry condition) {
+  private static void condition(List<String> textRecord, Entry condition, Boolean end) {
```

Comment:
One last request before we merge this - for these methods where you've added a new `end` parameter please add a description of the parameter to the javadoc above the method.